src/org/ods/OdsContext.groovy

```diff
@@ -303,10 +311,50 @@ class OdsContext implements Context {
       this.environmentCreated = created
   }
 
+  boolean getCiSkipEnabled() {
+    return (config.ciSkipEnabled == null || config.ciSkipEnabled)
+  }
+
+  def setCiSkipEnabled(boolean ciSkipEnabled) {
+    config.ciSkipEnabled = ciSkipEnabled
+  }
+
+  boolean getBitbucketNotificationEnabled() {
+    return (config.bitbucketNotificationEnabled == null || config.bitbucketNotificationEnabled)
+  }
+
+  def setBitbucketNotificationEnabled(boolean bitbucketNotificationEnabled) {
+    config.bitbucketNotificationEnabled = bitbucketNotificationEnabled
+  }
+
+  boolean getLocalCheckoutEnabled() {
+    return (config.localCheckoutEnabled == null || config.localCheckoutEnabled)
+  }
+
+  def setLocalCheckoutEnabled(boolean localCheckoutEnabled) {
+    config.localCheckoutEnabled = localCheckoutEnabled
+  }
+
+  boolean getDisplayNameUpdateEnabled() {
+    return (config.displayNameUpdateEnabled == null || config.displayNameUpdateEnabled)
+  }
+
+  def setDisplayNameUpdateEnabled(boolean displayNameUpdateEnabled) {
+    config.displayNameUpdateEnabled = displayNameUpdateEnabled
+  }
+
+  boolean getMailNotificationEnabled() {
+    return (config.mailNotificationEnabled == null) || config.mailNotificationEnabled
+  }
+
+  def setMailNotificationEnabled(boolean mailNotificationEnabled) {
+    config.mailNotificationEnabled = mailNotificationEnabled
+  }
+
   private String retrieveGitUrl() {
     script.sh(
       returnStdout: true, script: 'git config --get remote.origin.url'
-    ).trim().replace('https://bitbucket', 'https://cd_user@bitbucket')
+    ).trim().replaceAll('^(https?)://') { it[0] + 'cd_user@' }
```

Comment:
Maybe we should make the protocol and the user name configurable.

Reply:
Forget, what I said about matching http.

Reply:
Fixed. Username is loaded from credentials.